docs/usage.md

```diff
@@ -53,10 +53,18 @@ sample2,run1,ILLUMINA,sample2.unmapped_1.fastq.gz,sample2.unmapped_2.fastq.gz,sa
 
 ## Running the pipeline
 
-The typical command for running the pipeline is as follows:
+The example commands for running each workflow are as follows:
 
 ```bash
-nextflow run genomic-medicine-sweden/meta-val --input ./samplesheet.csv --outdir ./results -profile docker
+# Green Workflow - pathogen screening
+nextflow run genomic-medicine-sweden/meta-val --input ./samplesheet.csv --outdir ./results -profile docker --perform_screen_pathogens --pathogens_genomes /path/to/reference.fna --accession2taxid /path/to/accession2taxid.map
```

Comment:
Is the `accession2taxid` custom or is it downloaded from NCBI?

Reply:
accessions and taxid of pathogens. Users should prepare it. I have documented it in schema. But I could state that in usage again.

Reply:
Let's make it clear in the usage and you can resolve afterwards.

Reply:
I have added it to the usage.